src/app/route/CheckoutPage/CheckoutPage.component.js

```diff
@@ -0,0 +1,625 @@
+                                    onClick={ () => this.setState({ checkoutStep: CHECKOUT_STEP_REVIEW_AND_PAYMENTS }) }>
+                                    Next step
+                                </button>
+                            </div>
+
+
+                        ) : (
+                            <div block="checkoutStep" elem="reviewAndPayments">
+                                <fieldset block="CheckoutStep" elem="legend">
+                                    <legend>Payment Method</legend>
+                                    <table block="CheckoutStep" elem="OptionsTable">
+                                        <tbody>
+                                        { Object.keys(paymentMethods).map((index) => {
+                                            return this.renderPaymentMethod(index, paymentMethods[index]);
+                                        }) }
+                                        </tbody>
+                                    </table>
+                                </fieldset>
+
+                                <fieldset block="CheckoutStep" elem="legend">
+                                    <legend>Billing Address</legend>
+
+                                    <Field
+                                        id="sameAsShippingAddress"
+                                        type="checkbox"
+                                        block="billingAddress"
+                                        elem="sameAsShippingAddressCheckbox"
+                                        label="My billing and shipping address are the same"
+                                        value={ billingIsSame }
+                                        checked={ billingIsSame }
+                                        onChange={ () => this.setState({ billingIsSame: !billingIsSame }) }
+                                    />
+                                </fieldset>
+
+                                    { billingIsSame ? (
+                                        <address>
+                                            { `${ shippingAddress.firstname } ${ shippingAddress.lastname }` }<br/>
+                                            { shippingAddress.company }<br/>
+                                            { shippingAddress.street[0] }<br/>
+                                            { shippingAddress.street[1] }<br/>
+                                            { shippingAddress.city }<br/>
+                                            { shippingAddress.state }<br/>
+                                            { shippingAddress.zip }<br/>
+                                            { shippingAddress.country }<br/>
+                                            { shippingAddress.phone }<br/>
+                                        </address>
+                                    ) : (
+                                        <div>
+                                            <fieldset block="CheckoutStep" elem="legend">
```

Comment:
Please do not repeat yourself you have almost the same logic in shipping address merge this logic together, please.